mycore-base/src/main/java/org/mycore/frontend/cli/MCRObjectCommands.java

```diff
@@ -1358,6 +1358,20 @@ public static List<String> repairMetadataSearchForBase(String baseID) {
             .collect(Collectors.toList());
     }
 
+    @MCRCommand(
+            syntax = "repair shared metadata for the ID {0}",
+            help = "Retrieves the MCRObject with the MCRObjectID {0} and repairs the shared metadata.",
+            order = 172)
+    public static void repairInheritedMetadata(String id) throws MCRAccessException {
```

Comment:
syntax = "repair **shared** metadata ..." / method name = "repair **Inherited** Metadata"

I'am confused ;-)

Reply:
Changed it to repairSharedMetadata